cypress/integration/tests/developer/controls/jobfunctions/pagination.test.ts

```diff
@@ -104,7 +96,6 @@ describe("Job functions pagination validations", { tags: "@tier3" }, function ()
     it("Items per page validations", function () {
         // Navigate to Job functions tab
         Jobfunctions.openList();
-        cy.wait("@getJobfunctions");
```

Comment:
@abrugaro , is there any reason why this statement is being removed  after Jobfunctions.openList();  ?

Reply:
Sure @nachandr,

When you run the suite, it starts on the login page and enters the 'jobfunctions' for the first time. It makes sense to wait until the job functions are fetched from the API because it's the first time the user is accessing this specific menu.

On the other hand, the rest of the tests will start on that page, so a request may or may not occur (depending on the polling frequency) while we are waiting for it.